bestpy/best.py

```diff
@@ -0,0 +1,12 @@
+from .answers import answers
+
+import random
+
+
+class Best:
+    def __getattribute__(self, name):
+        # Return random value in list
+        if type(answers[name]) != list:
+            return answers[name]
+        else:
+            return random.choice(answers[name])
```

Comment:
using isinstance and swapping around the if and else makes more sense here.

```suggestion
        if isinstance(answers[name], list):
            return random.choice(answers[name])
        else:
            return answers[name]
```